src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+
+  &__item {
+    background-image: url(../images/star.svg);
+    margin-right: 4px;
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center center;
+  }
+
+  &:last-child {
+    margin-right: 0;
+  }
+}
+
+.stars__stars--0 .stars__item:nth-child(0),
+.stars__stars--1 .stars__item:nth-child(-n+1),
+.stars__stars--2 .stars__item:nth-child(-n+2),
+.stars__stars--3 .stars__item:nth-child(-n+3),
+.stars__stars--4 .stars__item:nth-child(-n+4),
+.stars__stars--5 .stars__item:nth-child(-n+5) {
+  background-image: url("../images/star-active.svg");
+}
```

Comment:
you can sass loop through with [@for](https://sass-lang.com/documentation/at-rules/control/for/)